plaso/filters/parser_filter.py

```diff
@@ -8,14 +8,16 @@ class ParserFilterExpressionHelper(object):
   """Helper for parser and plugin filter expressions.
 
   A parser filter expression is a comma separated value string that denotes
-  a list of parser names to include and/or exclude. Each entry can have
-  the value of:
-
-  * An exact match of a preset, which is a predefined list of parsers
-    and/or plugins (see data/presets.yaml for the list of predefined
-    presets).
-  * A name of a single parser (case insensitive), such as msiecf.
-  * A glob name for a single parser, such as '*msie*' (case insensitive).
+  which parsers should be used. Each value can contain either:
+
+  * The name of a preset (case sensitive), which is a predefined list of parsers
+    and/or plugins (see data/presets.yaml for the default presets).
+  * The name of a parser (case insensitive), for example 'msiecf'.
+  * The name of a plugin, prefixed with the parser name and a '/', for example
+    'sqlite/chrome_history'.
+
+  If the value begins with a '!' character the item will be excluded from the
```

Comment:
value => element

so that wording is consistent, with that you preferred in your review of the previous changes to this code